src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Badging for app icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-10-08
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
+alt: Phone showing several notification badges
+---
+
+{% Aside 'caution' %}
+  This API is part of the new
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and starting in Chrome 73 it is available as an [**origin trial**](#ot).
```

Comment:
Are any of the capabilities things also being worked on by other browsers? As much as possible we'd like to keep the site browser agnostic, so if there are opportunities to link to, or highlight progress by other browsers on these same APIs that would be good.

Reply:
For the most part Fugu APIs will be Chrome only until passed the OT state. That may change in the future once Edge has a way to do OTs, but for now they don't and are unable to enable these kinds of things.